### PR TITLE
Introduce rpc function to modify Project node

### PR DESCRIPTION
**Pull request by GiasemiSh**

### CODE REVIEW

File: buildservice.proto [hunk 1]
```diff
@@ -30,6 +30,7 @@ service BuildService {
   rpc Package(stream FileNode) returns (BuildResponse) {} 
   rpc CreatePackage(PackageNode) returns (BuildResponse) {}
   rpc CreateProject(ProjectNode) returns (BuildResponse) {}
+  rpc Project(stream PackageNode) returns (BuildResponse) {}
```
Comment (hemarkus) [inline buildservice.proto]: Chose a name that uses a verb to reflect what this function does.

**Review by hemarkus — CHANGES_REQUESTED** [review]
Change function name

File: buildservice.proto (hunk 1 above)
Comment (GiasemiSh) [reply buildservice.proto]: @hemarkus 
Ok. But we have to follow a generic syntax. I was trying to follow the structure that we use until now. Either I could name it Project (see Package function above) or SendProjectNode. But I chose UpdateProjectNode. I believe is more appropriate and we should follow the 'Update...Node' structure in the future.

**Review by hemarkus — APPROVED** [review]
lgtm

**Pull request merged**